packages/microcosm-www-next/src/pages/index.js

```diff
@@ -1,40 +1,152 @@
 import React from 'react'
+import data from '../data/index.json'
+import { Graphic1, Graphic2, Graphic3 } from '../components'
 
-const IndexPage = () => (
-  <section className="section">
-    <div className="wrapper">
-      <h2 className="section-title">
-        <span>01.</span>
-        Rendering a View
-      </h2>
-
-      <div className="figure-container">
-        <figure className="figure -left">
-          <img src="" className="figure__graphic" alt="TODO" />
-          <figcaption className="figure__content">
-            <h3 className="figure__content__header">In the browser</h3>
-            <p className="figure__content__text">
-              A user fires up the browser to take a quiz.
-            </p>
-          </figcaption>
-        </figure>
-
-        <figure className="figure -right">
-          <img src="" className="figure__graphic" alt="TODO" />
-          <figcaption className="figure__content">
-            <h3 className="figure__content__header">In Microcosm</h3>
-            <p className="figure__content__text">
+export default class IndexPage extends React.Component {
+  constructor(props) {
+    super(props)
+  }
```

Comment:
This isn't necessary. By default, a class calls the constructor of its ancestor if it isn't specified.